### DISCUSSION TITLE
can Hive send email notification when an alert is escalated or a case is created?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#4757] [Unsupported Versions]
I have a team that reviews alerts and escalates suspicious stuff.  Currently they Slack message me to inform that they escalated something.  Is there a way for the Hive to send an auto notification email when an escalation occurs?  I'd like to use a Slack channel address as the recipient.

**Comment by weslambert** [comment]
You've got a couple options I can think off of the top of my head:

(1) Configure an Elastalert rule that looks for `event.escalated: true`, with an output for Email or Slack (and other rule options to aggregate similar results, make it more reliable).

(2) Subscribe to TheHive's webhook notifications using an HTTP listener, then call the mailer responder, or submit an email (or Slack webhook POST) from a script or other tool.
